backend/cmd/jobs/update-handler.go

```diff
@@ -0,0 +1,61 @@
+package main
+
+import (
+	"context"
+	"github.com/bcc-code/brunstadtv/backend/common"
+	"github.com/bcc-code/brunstadtv/backend/push"
+	"github.com/bcc-code/brunstadtv/backend/scheduler"
+	"github.com/bcc-code/brunstadtv/backend/sqlc"
+	"github.com/bcc-code/brunstadtv/backend/utils"
+	"github.com/bcc-code/mediabank-bridge/log"
+	"github.com/go-redsync/redsync/v4"
+	"github.com/google/uuid"
+	"time"
+)
+
+type modelHandler struct {
+	queries   *sqlc.Queries
+	push      *push.Service
+	scheduler *scheduler.Service
+	locker    *redsync.Redsync
+}
+
+func (h *modelHandler) handleModelUpdate(ctx context.Context, collection string, key string) error {
+	switch collection {
+	case "notifications":
+		id := uuid.MustParse(key)
```

Comment:
Not sure if this is the best here. `must` by convention panics so if I manage to sneak something that's not UUID in, I can get instance to crash. Do that continuously and we have a DOS.